modules/dicom_archive/php/dicom_archive.class.inc

```diff
@@ -48,6 +48,9 @@ class Dicom_Archive extends \NDB_Menu_Filter
      */
     function _setupVariables()
     {
+        // Rest the filter because this is broken with react modules
+        $this->_resetFilters();
```

Comment:
How is this going to work if you explicitly set a filter on the dicom archive page itself and each page load resets the filters?